setup/Tycho.setup

```diff
@@ -32,8 +32,8 @@
       <setupTask
           xsi:type="setup:PreferenceTask"
           key="/instance/org.eclipse.egit.ui/commit_dialog_signed_off_by"
-          value="true">
-        <description>Have egit add the signed-off-by footer to the commit message by default</description>
+          value="false">
+        <description>Signed-off-by not used anymore with github</description>
```

Comment:
It's not with GitHub, it's the Eclipse Development Process that was simplified, so it applies to whichever repository and codebase.

Reply:
Thanks for clarifying. I wasn't aware of that and thought it is based on whether gerrit or github is used.